tsconfig.json

```diff
@@ -1,18 +1,11 @@
 {
+  "extends": "@tsconfig/node16/tsconfig.json",
   "compilerOptions": {
-    "target": "esnext",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": ["dom", "dom.iterable", "es2021"],
     "allowJs": true,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
```

Comment:
Id prefer to keep strict mode enabled, but am okay with turning it off temporarily to get this up and running.

Reply:
I removed `strict` here because it's already included in the preset being extended